pxf/pxf-jdbc/src/main/java/org/apache/hawq/pxf/plugins/jdbc/JdbcPlugin.java

```diff
@@ -58,8 +58,8 @@ public JdbcPlugin(InputData input) throws UserDataException {
         super(input);
         jdbcDriver = input.getUserProperty("JDBC_DRIVER");
         dbUrl = input.getUserProperty("DB_URL");
-        user = input.getUserProperty("USER");
-        pass = input.getUserProperty("PASS");
+        user = input.getUserProperty("DB_USER");
+        pass = input.getUserProperty("DB_PASS");

```

Comment:
no longer necessary to change these property names

Reply:
yes ... we don't need this